ods_ci/tests/Tests/100__deploy/120__upgrades/121__during_upgrade.robot

```diff
@@ -85,3 +88,12 @@ Upgrade Test Teardown
     Log    rhods_aggregate_availability: ${resp.json()["data"]["result"][0]["value"][-1]}
     @{list_values} =    Create List    1
     Run Keyword And Warn On Failure    Should Contain    ${list_values}    ${resp.json()["data"]["result"][0]["value"][-1]}
+
+Is RHODS Version Greater Than
+    [Documentation]    Returns True if:
+    ...    - RHODS version is greater or equal than ${initial_version}
+    [Arguments]  ${initial_version}
+    ${ver} =  Get RHODS Version
+    ${ver} =  Fetch From Left  ${ver}  -
```

Comment:
Does this work properly with z.stream too? E.g. 2.8.3?

Reply:
I just tested it and it works with z-streams, I could upgrade from 2.9.0 to 2.9.1 using this test. ${RHODS_VERSION} variable is X.Y.Z.